Sources/SwiftUICharts/LineChart/FullScreenLineChartView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  FullScreenLineChartView.swift
+//
+//
+//  Created by Joshua Holme on 2/9/21.
+//
+
+import SwiftUI
+
+public struct FullScreenLineChartView: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
+    @ObservedObject var data:ChartData
+    public var title: String
+    public var legend: String?
+    public var style: ChartStyle
+    public var darkModeStyle: ChartStyle
+
+    public var dropShadow: Bool
+    public var valueSpecifier:String
+    
+    @State private var touchLocation:CGPoint = .zero
+    @State private var showIndicatorDot: Bool = false
+    private var rateValue: Int?
+    
+    public init(data: [Double],
+                title: String,
+                legend: String? = nil,
+                style: ChartStyle = Styles.lineChartStyleOne,
+                rateValue: Int? = nil,
+                dropShadow: Bool? = true,
+                valueSpecifier: String? = "%.1f") {
+        
+        self.data = ChartData(points: data)
+        self.title = title
+        self.legend = legend
+        self.style = style
+        self.darkModeStyle = style.darkModeStyle != nil ? style.darkModeStyle! : Styles.lineViewDarkMode
```

Comment:
this could be simplified to `style.darkModeStyle ?? Styles.lineViewDarkMode`

Reply:
This code was copied from elsewhere in the package. Updated in the next commit.